api/proto/config/devconfig.proto

```diff
@@ -56,6 +57,11 @@ message EdgeDevConfig {
         // Information saved by device to make it easier to find in the controller
         string enterprise = 17;
         string name = 18;
+
+        // Controller supplies a list of cipher contexts
```

Comment:
It's a reasonable starting point to include it in EdgeDevConfig, but I suspect we want to split it out since the certificate(s) are large and don't change much. As long as it can stand alone and come as part of the certs API that is fine.

Reply:
Sure.